test/bdd/features/edv_e2e_api.feature

```diff
@@ -8,17 +8,11 @@
 @edv_rest
 Feature: Using EDV REST API
 
-  @create_new_vault
-  Scenario: Client creates a new data vault
+  @e2e
+  Scenario: Full end-to-end flow. Create a data vault, store an encrypted document, and then retrieve the encrypted document.
     Then  Client sends request to create a new data vault with id "testvault" and receives the vault location "localhost:8080/encrypted-data-vaults/testvault" in response
-
-  @create_new_document
-  Scenario: Client creates a new document
-    Given EDV server has a data vault with id "testvault2"
-    Then  Client sends request to create a new document with id "VJYHHJx4C8J9Fsgz7rZqSp" in the data vault with id "testvault2" and receives the document location "localhost:8080/encrypted-data-vaults/testvault2/docs/VJYHHJx4C8J9Fsgz7rZqSp" in response
-
-  @read_document
-  Scenario: Client retrieves a previously stored document
-    Given EDV server has a data vault with id "testvault3"
-    Given The data vault with id "testvault3" has a document with id "VJYHHJx4C8J9Fsgz7rZqSp"
-    Then  Client sends request to retrieve the previously stored document with id "VJYHHJx4C8J9Fsgz7rZqSp" in the data vault with id "testvault3" and receives the document "${EXPECTED_DOCUMENT}" in response
\ No newline at end of file
+    Then  Client constructs a Structured Document with id "VJYHHJx4C8J9Fsgz7rZqSp"
```

Comment:
Do we need those two steps in BDD? Seems like a detail.

Reply:
I include those as separate steps since I feel they're important parts of the EDV flow that I think are not obvious. Specifically that the entire Structured Document is actually supposed to be encrypted and then stuffed into the 'JWE' field. The BDD test is really the only way to know that you're supposed to do that (unless you look at the spec).

If you still think it doesn't need to be a separate step, then let me know and I'll merge them into the "store document" step.